tests/test_EvaluateLocalCalibration.py

```diff
@@ -53,6 +53,16 @@ def testPhotoCalib(self):
         self.assertEqual(record.get("base_LocalPhotoCalib"), calib)
         self.assertEqual(record.get("base_LocalPhotoCalibErr"), calibErr)
 
+    def testPhotoCalibIsNone(self):
+        task = self.makeSingleFrameMeasurementTask("base_LocalPhotoCalib")
+        exposure, catalog = self.dataset.realize(10.0, task.schema, randomSeed=0)
+        exposure.setPhotoCalib(None)
+        task.run(catalog, exposure)
```

Comment:
To confirm that things work as expected, you should also wrap this in a log checker, e.g.:
```
import logging  # at top

with self.assertNoLogs(level=logging.WARNING):
    task.run(catalog, exposure)
```